### ISSUE TITLE
PowerShell can't get security protocol set in fetch-thirdparty-deps.ps1

### ISSUE THREAD

**Issue by Grovestreetgman** [OpenRA/OpenRA#15052]
Attempting to compile the latest bleed build, and this occurs:

![ttfuql2](https://user-images.githubusercontent.com/34424247/38725738-468e88a8-3ed6-11e8-9ac9-a02323383aca.png)

The first exception is the most telling it seems. I changed "Tls12" to "Tls" in fetch-thirdparty-deps.ps1 just to test it, and it was able to connect to MaxMind.

![jkki8xr](https://user-images.githubusercontent.com/34424247/38725745-4cf897a6-3ed6-11e8-882e-53f4498cc459.png)

From this, I would assume that since it can't parse "Tls12", it is attempting to connect to Github using TLS 1.1 instead of 1.2, which would lead to the three proceeding errors (see pchote's post on #14884 ).

Related to #14855 

I am running .NET 4.7.1 on Windows 7.

**Comment by abcdefg30** [comment]
What is your powershell version? Updating that might help.

**Comment by Grovestreetgman** [comment]
According to Powershell's own version command, I am running 2.0, as shipped with windows 7 service pack 1. It seems the only way to upgrade past this is to install Windows Management Framework, which I don't believe windows update does. This may need to be listed as a user-installed requirement. I will test to see if it fixes the problem and post my findings.

**Comment by Grovestreetgman** [comment]
Got it working. I had to install Windows Management Framework. I initially tried installing 5.1, but it would not install, even after installing 4.0; Microsoft says it's compatible with Windows 7, but the installer says otherwise. Doing so upgrades Powershell to the same version number. I did not test 3.0, as Windows 8.0 is the only system that ships with it (8.1 upgrades it to 4.0), but I would assume 4.0 and everything above would fix the issue, provided it is compatible. 

May want to list Windows Management Framework 4.0 as required for Windows 7 users if the user does not have/want to install Visual studio to compile the builds.
